core/src/main/java/io/gnosis/kouban/core/utils/IconUtils.kt

```diff
@@ -31,6 +33,28 @@ fun ImageView.setTransactionIcon(picasso: Picasso, icon: String?) {
     }
 }
 
+fun RemoteViews.setTransactionIcon(@IdRes imageViewId: Int, picasso: Picasso, icon: String?, appWidgetIds: IntArray) {
+    setViewPadding(imageViewId, 0, 0, 0, 0)
```

Comment:
This is a lot of code duplication, in a production app I would try to refactor this

Reply:
yes, not important right now